pkg/util/tags.go

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we place this into `pkg/runtime/tags.go` please?

Reply:
Moved to `pkg/runtime/tags.go`